mathesar/rpc/columns.py

```diff
@@ -74,6 +74,28 @@ def from_dict(cls, col_default):
             )
 
 
+class CreateableColumnInfo(TypedDict):
```

Comment:
At some point, we need to figure out a way to use inheritance here in a way that preserves the documentary value of these classes.